portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,141 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query;
+import java.util.*;
+
+/** Represents a datastore of comments.
+ * @author Olga Shimanskaia
+ * @author oshimanskaia@google.com
+ * @version 2.2.0
+ * @since 2.2.0
+ */
+public class Comments {
+    private final String ENTITY_KIND = "Comment";
+    public static final String DATE = "date";
+    public static final String RATING = "rating";
+    public static final String TEXT = "comment-text";
+
+    /** Represents the comment.
+     */
+    private static class Comment {
+        Date date;
+        String text;
+        long rating;
+
+        /** Creates the comment with specified fields.
+         * @param date      The date of sending a comment.
+         * @param text      The comment's text.
+         * @param rating    The comment's rating.
+         */
+        public Comment(Date date, String text, long rating) {
+            this.date = date;
+            this.text = text;
+            this.rating = rating;
+        }
+    }
+
+    private final DatastoreService datastore;
+
+    public Comments() {
+        this.datastore = DatastoreServiceFactory.getDatastoreService();
+    }
+
+    /** Creates the comment with the specified text and puts it to the datastore.
+     * @param text      The comment's text.
+     */
+    public void addComment(String text) {
+        Entity commentEntity = new Entity(ENTITY_KIND);
+        commentEntity.setProperty(DATE, new Date());
+        commentEntity.setProperty(TEXT, text);
+        commentEntity.setProperty(RATING, 0);
+
+        datastore.put(commentEntity);
+    }
+
+    /** Gets the employee’s last name.
```

Comment:
The documentation doesn't match what the method returns

Reply:
ahahah, I just saw the example of how to write javadoc for java classes and forgot to change this example, funny